File: frontend/src/types/brief.ts
export interface LumaConcept {
  label: string;
  hook_type: string;
  luma_prompt: string;
}

export interface CreativeBrief {
  ad_id: string;
  title: string;
  hook_type: string;
  format_length: string;
  visual_pacing: string;
  creative_direction: string;
  luma_prompt: string;
  performance_rationale: string;
  concepts: LumaConcept[];
}

File: backend/app/schemas/brief.py
from pydantic import BaseModel, Field


class LumaConcept(BaseModel):
    label: str
    hook_type: str
    luma_prompt: str


class CreativeBrief(BaseModel):
    ad_id: str
    title: str
    hook_type: str
    format_length: str
    visual_pacing: str
    creative_direction: str
    luma_prompt: str
    performance_rationale: str
    concepts: list[LumaConcept] = Field(default_factory=list)
